shell-completion/zsh/_machinectl

```diff
@@ -1,20 +1,5 @@
 #compdef machinectl
 
-__get_available_machines () {
-    machinectl --no-legend list-images | {while read -r a b; do echo $a; done;}
-}
-
-_available_machines() {
-    local -a _machines
-    _machines=("${(fo)$(__get_available_machines)}")
-    typeset -U _machines
-    if [[ -n "$_machines" ]]; then
-        _describe 'machines' _machines
-    else
-        _message 'no machines'
-    fi
-}
-
```

Comment:
That zsh completion syntax is impenetrable. I've written a bunch of it and every time I feel like it's been ROT13-ed.

Can you explain in the commit message what is changed and why? What is "-t i"?